index.html

```diff
@@ -10,16 +10,41 @@
 </head>
 
 <body>
-  <h1>Awesome Books</h1>
-  <section>
-    <ul>
+    <section class = "nav-menu">
+        <nav class="navbar">
+         <h1 class="title">Awesome Books</h1>
+         <ul class="nav-list">
+             <li id ="list" class="list"> List</li>
+              <li id= "new" class="new"> Add-New</li>
```

Comment:
- [ ] [OPTIONAL] Kindly let's ensure that the navbar item is well styled by giving it the necessary spaces as it is in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements).
![Screenshot from 2022-05-27 21-24-31](https://user-images.githubusercontent.com/58692590/170785984-dacb3f85-9c53-4710-8458-be485deac22d.png)